packages/webdoc-default-template/tmpl/components/member/params.tmpl

```diff
@@ -0,0 +1,57 @@
+<?js
+const params = obj;
```

Comment:
Again, the description on top.

Reply:
Fixed in 9e7db62055baad1b7af9e1bcbdbfbd3fa02c5520.